scripts/vm/systemvm/injectkeys.sh

```diff
@@ -24,14 +24,8 @@
 set -e
 
 TMP=/tmp
```

Comment:
@Pearl1594 should this file be completely removed if systemvm.iso isn't used? (unless this used for some other purposes too)?

Reply:
This script served 2 purposes - 

1. to copy the private key to the hosts 
2. inject ssh keys into the systemvm iso

While operation defined in point 2 is no longer required to be performed, copying private key is still required